tests/test_config_parsing/test_num_cpu.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from ert._c_wrappers.enkf import ConfigKeys, EnKFMain, ResConfig
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_default_num_cpu():
+    config_dict = {ConfigKeys.NUM_REALIZATIONS: 1}
+    res_config = ResConfig(config_dict=config_dict)
+    enkf_main = EnKFMain(res_config)
+    assert 1 == enkf_main.get_num_cpu()
```

Comment:
a bit pedantic maybe, but since we are introducing new code and having this github linting in place i guess we could try to adhere to the standards , i.e enkf_main.get_num_cpu() == 1 😃